content/Video_Display.md

```diff
@@ -127,9 +127,9 @@ drawing exceeds the lower (right) border of the BG map area.
 
 ### FF44 - LY (LCDC Y-Coordinate) (R)
 
-The LY indicates the vertical line to which the present data is
-transferred to the LCD Driver. The LY can take on any value between 0
-through 153. The values between 144 and 153 indicate the V-Blank period.
+The LY indicates the horizontal line to which the present data is
```

Comment:
I'm not sure "to which the present data [...]" is very clear... how should it be reworded? Talking about "the line being currently drawn" may be misleading, since no drawing occurs during HBlank or Mode 2...

Reply:
Hmm, I'm not sure about the current description. Maybe this: "LY indicates the current horizontal line, which might be about to be drawn, being drawn, or just been drawn."?

Reply:
Yes, I think that's fine. Fixed.